src/index.ts

```diff
@@ -20,43 +21,25 @@ class PinchPlugin extends BasePlugin {
     this.viewport = options.viewport;
     this.listenerNode = options.listenerNode || document.body;
 
-    // A bit of silly type coercion here to override TypeScript, which is smart
-    // enough to know that both the event names and objects are totally invalid.
-    this.listenerNode.addEventListener(
-      'gesturestart' as any,
-      this.onGestureStart as any
-    );
-    this.listenerNode.addEventListener(
-      'gesturechange' as any,
-      this.onGestureChange as any
-    );
-    this.listenerNode.addEventListener(
-      'gestureend' as any,
-      this.onGestureEnd as any
-    );
+    this.listenerNode.addEventListener('gesturestart', this.onGestureStart);
+    this.listenerNode.addEventListener('gesturechange', this.onGestureChange);
+    this.listenerNode.addEventListener('gestureend', this.onGestureEnd);
 
-    this.initialScale = 1;
+    this.initialScale = this.viewport.scale.x;
```

Comment:
Using just `x` seems a little questionable (though we might be doing this somewhere else? I forget?). Is it possible to scale X and Y by different values?

Reply:
We are doing it when we set it in `gesturestart`. It _is_ possible to set the scale differently, but we don't. If we really want to be safe, we'd save both scales and scale `x` and `y` by those scales. Probably a good follow up.

Reply:
Sounds good. Definitely not blocking.